src/opm/output/eclipse/Summary.cpp

```diff
@@ -682,6 +682,23 @@ inline quantity injection_history( const fn_args& args ) {
     return { sum, rate_unit< phase >() };
 }
 
+inline quantity abondoned_wells( const fn_args& args ) {
+    std::size_t count = 0;
+
+    for (const auto sched_well : args.schedule_wells) {
+        if (sched_well.hasProduced()) {
+            const auto& well_name = sched_well.name();
+            if (args.wells.count(well_name)) {
+                auto output_well = args.wells.at( sched_well.name() );
+                if (!output_well.flowing())
+                    count++;
+            }
+        }
```

Comment:
Technically, this looks up the well twice.  It's not a huge cost, but we could consider writing this loop body as
```C++
if (!sched_well.hasProduced())
    continue;

auto xwPos = args.wells.find(sched_well.name());
if (xwPos == args.wells.end())
    continue;

count += !xwPos->second.flowing();
```

Reply:
OK - I have rewritten to use an iterator.